### ISSUE TITLE
punishment mode (horny mode)

### ISSUE THREAD

**Issue by Aproxia-dev** [Gankra/cargo-mommy#5]
mommy WILL threaten to punish you if your code won't compile

**Comment by Johnystar** [comment]
bruh

**Comment by VixieTSQ** [comment]
perhaps something like...
```rust
struct Responses {
    positive: Vec<Response>,
    negative: Vec<Response>,
}

struct Response {
    content: String,
    attributes: Vec<Attribute>
}

#[non_exhaustive]
enum Attribute {
    NSFW,
    Degrading
}
```

**Comment by cristeigabriel** [comment]
> Blessed are those who wash their robes, so that they may have the right to the tree of life and that they may enter the city by the gates. Outside are the dogs and sorcerers and the sexually immoral and murderers and idolaters, and everyone who loves and practices falsehood
Revelation 22:14

**Comment by thedenisnikulin** [comment]
this is getting out of control

**Comment by Aproxia-dev** [comment]
> this is getting out of control

@thedenisnikulin good.

**Comment by Johnystar** [comment]
any updates on this? this feature would really be super useful for me, so i was wondering, if anyone's working on this
i could probs also give it a look, but my exp with rust is kinda not much :))))
cheers

**Comment by VixieTSQ** [comment]
> any updates on this? this feature would really be super useful for me, so i was wondering, if anyone's working on this i could probs also give it a look, but my exp with rust is kinda not much :)))) cheers

I would be happy to work on this if @Gankra is comfortable with it. I really would like to wait for the go ahead from her, though... and even then, I would wait for #13 to get merged.

**Comment by blyxyas** [comment]
Really looking towards this in the tech mommy landscape for 2023. Can it use blockchain and gpt3???

**Comment by blyxyas** [comment]
> And yet indeed she is my sister; she is the daughter of my tech daddy, but not the daughter of my tech mommy; and she became my waifu.

I find this fragment of Genesis 20:12 deeply motivating towards my soul. It reminds me of this crate, and I think this is the real motive to implement a **horny mode** into `cargo-mommy`. Because it is what God wants us to do.

**Comment by VixieTSQ** [comment]
> > And yet indeed she is my sister; she is the daughter of my tech daddy, but not the daughter of my tech mommy; and she became my waifu.
> 
> I find this fragment of Genesis 20:12 deeply motivating towards my soul. It reminds me of this crate, and I think this is the real motive to implement a **horny mode** into `cargo-mommy`. Because it is what God wants us to do.

Sadly still waiting on @Gankra approval :(

**Comment by blyxyas** [comment]
@Gankra we really need you in this very moment.

**Comment by blyxyas** [comment]
When we needed her the most, she didn't return :(

**Comment by Gankra** [comment]
FINE I WILL WRITE YOU SMUT

**Comment by VixieTSQ** [comment]
> FINE I WILL WRITE YOU SMUT

It's a miracle!

**Comment by Gankra** [comment]
shipped in 0.2.0, see README for docs

**Comment by Ciubix8513** [comment]
Omg, Christmas came early this year
